viya4-iac-aws-darksite/custom-ami/README.md

```diff
@@ -0,0 +1,159 @@
+## AWS Darksite Custom AMI
+By default, viya4-iac-aws uses the base Canonical Ubuntu Server image, which does not have the required nfs-kernel-server package.  IAC installs these packages as part of the cloud-init process, however this is not possible in a dark site which has no outbound internet access.  Below are instructions for creating a custom AMI which will ensure that nfs-kernel-server packages are installed for both the jumpserver and NFS server.
+
+Once the AMI is created, you can then apply the mod to viya4-iac-aws by running the [darksite-iac-aws-mods.sh](https://gitlab.sas.com/jocobu/viya4-aws-darksite/-/tree/main/viya4-iac-aws/darksite-iac-aws-mods/darksite-iac-aws-mods.sh) script.
+
+## Notes
+This information was provided by [Frederik Vandenberghe](https://gitlab.sas.com/sbxfrv).  
+
+## Launch EC2-instance based on the current AMI in use
+First thing to do is launch an EC2-instance based on the AMI that's in use as AWS-base image for the nfs-server and the jump-server.  You can see which AMI that is in https://github.com/sassoftware/viya4-iac-aws/blob/main/modules/aws_vm/main.tf
+
+![](img/img1.png)
```

Comment:
Guessing this is the code snippet that's an image. Is there a reason this is not plain text one could copy?

Reply:
@thpang, the main reason that I used the image is that the README content from Josh's repo and the inline images that he included is that they are all intended to work together to help illustrate the required steps for creating a custom AMI image as simply as possible. I suspect he thought that the images were more helpful than the words that might replace it. In some cases he highlights relevant fields or indicates the important parts of an AWS console page action with arrows. 

Would you consider us releasing this as is in the experimental phase since we know that it is a working set of instructions and following up with another task that tries to replace the existing images with text if that is what you would prefer we have instead?